### ISSUE TITLE
`<script>` tag HTML attributes not highlighted properly in `*.astro` files

### ISSUE THREAD

**Issue by adamjohnson** [Destiner/sublime-astro#3]
Here's a comparison between Sublime and VS Code:

## Sublime (busted)

![Sublime script tag with non-colorized attributes screenshot](https://user-images.githubusercontent.com/575865/148598575-c2f89fb7-dde8-488c-aefb-d0788abd983c.png)

## VS Code (proper highlighting)

![VS Code script tag with proper syntax highlighting](https://user-images.githubusercontent.com/575865/148598751-1baed14d-48db-4605-b3a3-a53676fdc7ec.png)

## Code examples

Here are some code examples to copy and paste for easier debugging:

```html
  <script src={Astro.resolve('../scripts/main.js')} hoist></script>
  <script src="example.js" type="module" defer="true"></script>
```

**Comment by deathaxe** [comment]
Fixed by Astro 2.0.0 (now on https://github.com/SublimeText/Astro)

**Comment by adamjohnson** [comment]
Works great, ⭐ 👏 .
